index.js

```diff
@@ -18,20 +16,25 @@ app.listen(PORT, function(req, res){
 	console.log(`Server is running @ ${PORT}`);
 })
 
-
-
 app.get('/getWithYear', malaria);
 
-
 function malaria(req, res){
-	request(url, function(err, response, body){
+		var options ={
+			query: req.query.query,
+			date:`( FIRST_PDATE:[${req.query.start} TO ${req.query.end}])`,
+			sort: 'CITED desc',
+			pageSize:1,
+			synonym: true,
+			format: 'json'
+		}
+		request({url:reqUrl, qs:options }, function(err, response, body){
+			console.log(options)
 		if(!err && response.statusCode == 200){
 			var data = JSON.parse(body);
-			var malaria= []
-			data.resultList.result.forEach(function(Result) {
-  				malaria.push({Title: Result.title, Author: Result.authorString, Year: Result.pubYear, Count: Result.citedByCount});  				
-			});
-			res.status(200).json(malaria);
+				data.resultList.result.forEach(function(result) {
+  				var responseData = {title: result.title, author: result.authorString, pubYear: result.pubYear, count: result.citedByCount};
+  				res.send(responseData);  				
```

Comment:
@Firnaa5 when you push your code please make sure it is working. This will throw an error you cannot do res.send inside a forLoop. You need to think of something clever to handle the data here. basically I need the response to be something like this,
```
{
2016: { title: 'Title1', author: 'author1', year: '2016', count: 'xxx'},
2017: { title: 'Title1', author: 'author1', year: '2017', count: 'xxx'},
2018: { title: 'Title1', author: 'author1', year: '2018', count: 'xxx'}
}
```
please rethink the way you designed here.